arch/arm/core/aarch64/Kconfig

```diff
@@ -58,9 +58,6 @@ config CMSIS_V2_THREAD_MAX_STACK_SIZE
 config CMSIS_V2_THREAD_DYNAMIC_STACK_SIZE
```

Comment:
why is this being removed? I know it was used in A101, but there is nothing here that is related to 101 and we have a test. So I would leave it, we might get users for this in the future.

Reply:
@nashif Do you mean drop the commit 58e0c009ab5fb0df07d33fa365f655d1d0956b6f ?

Reply:
yes